<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseView</class>
 <widget class="QWidget" name="CaseView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>550</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelTitle">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Garamond</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 87, 86);</string>
         </property>
         <property name="text">
          <string>Titulo del caso actual </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnNewCase">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#btnNewCase:hover{
	background-color: rgb(218, 240, 240);
	border: 1px solid #1089a8;
	border-radius: 20px;
}

#btnNewCase:checked{
	background-color: rgb(180, 240, 240);
	border: 1px solid #1089a8;
	border-radius: 20px;
}
</string>
         </property>
         <property name="icon">
          <iconset resource="../../media.qrc">
           <normaloff>:/icons/Resources/imgs/icons/setDefault/Add File-32.png</normaloff>:/icons/Resources/imgs/icons/setDefault/Add File-32.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelNewCase">
         <property name="font">
          <font>
           <family>Garamond</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 87, 86);</string>
         </property>
         <property name="text">
          <string>Crear nuevo caso</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QWidget" name="inputSearch" native="true">
             <property name="minimumSize">
              <size>
               <width>235</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>235</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white; 
border: 1px solid rgb(156,208,208);
border-radius: 6px;</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="labelIconSearch">
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../media.qrc">:/icons/Resources/imgs/icons/setDefault/Search Filled2-24.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditSearch">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Garamond</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none; 
color: rgb(56,87,86);</string>
                </property>
                <property name="maxLength">
                 <number>250</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>Tus Casos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listViewOwnCases">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>170</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>170</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Garamond</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QScrollBar:vertical {               
        border: 1px solid #9CD0D0;
        background:white;
        width:10px;    
        margin: 0px 0px 0px 0px;
    }
    QScrollBar::handle:vertical {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0  rgb(156,208,208), stop: 0.5 rgb(156,208,208),  stop:1 rgb(156,208,208));
        min-height: 0px;
    
    }
    QScrollBar::add-line:vertical {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0  rgb(156,208,208), stop: 0.5 rgb(156,208,208),  stop:1 rgb(156,208,208));
        height: px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
    }
    QScrollBar::sub-line:vertical {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0  rgb(156,208,208), stop: 0.5 rgb(156,208,208),  stop:1 rgb(156,208,2080));
        height: 0px;
        subcontrol-position: top;
        subcontrol-origin: margin;
    }

QListView{ 
	color: rgb(56,87,86);
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(156, 208, 208);
	border-radius: 3px }</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QWidget" name="inputSearchOthers" native="true">
             <property name="minimumSize">
              <size>
               <width>235</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>235</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white; 
border: 1px solid rgb(156,208,208);
border-radius: 6px;</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayoutOthers">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="labelIconSearchOthers">
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../media.qrc">:/icons/Resources/imgs/icons/setDefault/Search Filled2-24.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditSearchOthers">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Garamond</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none; 
color: rgb(56,87,86);</string>
                </property>
                <property name="maxLength">
                 <number>250</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>Otros Casos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listViewOtherCases">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>170</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>170</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Garamond</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QScrollBar:vertical {               
        border: 1px solid #9CD0D0;
        background:white;
        width:10px;    
        margin: 0px 0px 0px 0px;
    }
    QScrollBar::handle:vertical {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0  rgb(156,208,208), stop: 0.5 rgb(156,208,208),  stop:1 rgb(156,208,208));
        min-height: 0px;
    
    }
    QScrollBar::add-line:vertical {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0  rgb(156,208,208), stop: 0.5 rgb(156,208,208),  stop:1 rgb(156,208,208));
        height: px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
    }
    QScrollBar::sub-line:vertical {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0  rgb(156,208,208), stop: 0.5 rgb(156,208,208),  stop:1 rgb(156,208,2080));
        height: 0px;
        subcontrol-position: top;
        subcontrol-origin: margin;
    }

QListView{ 
	color: rgb(56,87,86);
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(156, 208, 208);
	border-radius: 3px }</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widgetChatView" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>170</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>170</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(254, 255, 164)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>550</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>550</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Garamond</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="iconSize">
        <size>
         <width>130</width>
         <height>48</height>
        </size>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tabCaseFaseView">
        <attribute name="icon">
         <iconset resource="../../media.qrc">
          <normaloff>:/icons/Resources/imgs/icons/setDefault/Checklist-48.png</normaloff>
          <normalon>:/icons/Resources/imgs/icons/setDefault/Checklist Filled-48.png</normalon>:/icons/Resources/imgs/icons/setDefault/Checklist-48.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabCaseDocView">
        <attribute name="icon">
         <iconset resource="../../media.qrc">
          <normaloff>:/icons/Resources/imgs/icons/setDefault/Box-48.png</normaloff>
          <normalon>:/icons/Resources/imgs/icons/setDefault/Box Filled-48.png</normalon>:/icons/Resources/imgs/icons/setDefault/Box-48.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabCostView">
        <attribute name="icon">
         <iconset resource="../../media.qrc">
          <normaloff>:/icons/Resources/imgs/icons/setDefault/Money Box-48.png</normaloff>
          <normalon>:/icons/Resources/imgs/icons/setDefault/Money Box Filled-48.png</normalon>:/icons/Resources/imgs/icons/setDefault/Money Box-48.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../media.qrc"/>
 </resources>
 <connections/>
</ui>
